src/webui/www/private/scripts/color-scheme.js

```diff
@@ -0,0 +1,28 @@
+"use strict";
```

Comment:
Please add license text here.

Reply:
Yeah about that, I honestly don't know what to put there.

Reply:
You can put the 'standard' license qbt uses:
https://github.com/qbittorrent/qBittorrent/blob/b462a2bf0cd2253cf24494a68af6c4773ced6a1c/src/app/filelogger.h#L1-L27
And put your or sledgehammer999 name in it.